### PR TITLE
Posix compat

### PR DESCRIPTION
**Pull request by ShellyHerself**
I still want to add a few touches after this. But this is most of it.

### CODE REVIEW

File: windows/tools/tag_scanner_window.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     # leaving this here just in case it somehow works though.
     TEXT_EDITOR_NAME = "TextEdit"
 elif platform == "linux":
```
Comment (ShellyHerself) [inline windows/tools/tag_scanner_window.py]: Might not work for certain Linux versions. Needs to be
```suggestion
elif "linux" in platform:
```

File: editor_constants.py [hunk 2]
```diff
@@ -33,5 +36,36 @@
...
+
+MOZZ_BITMAP_PATH = Path(MOZZLIB_DIR, "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = Path(MOZZLIB_DIR, "icons", "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = Path(WORKING_DIR, "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = Path(WORKING_DIR, "icons", "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = ""
```
Comment (ShellyHerself) [inline editor_constants.py:67]: If we consistently store the icons in the library folders then this is not needed.

**Review by ShellyHerself — COMMENTED** [review]
Some of my personal notes.

File: windows/tools/tag_scanner_window.py [hunk 3]
```diff
@@ -369,6 +358,7 @@ def scan(self):
 
                 except Exception:
                     print("    Could not scan '%s'" % tag.filepath)
+                    print(format_exc())
```
Comment (MosesofEgypt) [inline windows/tools/tag_scanner_window.py:357]: A print of the stack trace should always be before any "summary" statement. The user will see a big stack trace, and the thing they'll usually do is look at the bottom for the tl; dr

**Pull request merged**